driver/driver.feature

```diff
@@ -921,10 +917,27 @@ Feature: TypeDB Driver
     Then answer get row(0) get variable(p) is entity: false
     Then answer get row(0) get variable(p) is relation: false
     Then answer get row(0) get variable(p) is attribute: false
+    Then answer get row(0) get variable(p) is struct: false
+    Then answer get row(0) get variable(p) is boolean: false
 
     Then answer get row(0) get variable(p) as entity type
+    Then answer get row(0) get variable(p) try get label: person
     Then answer get row(0) get variable(p) get label: person
+    Then answer get row(0) get variable(p) try get iid is none
```

Comment:
Similar for `try get iid`, `try get value type`, `try get value`, `try get boolean`, etc.